WindowsServerDocs/manage/windows-admin-center/configure/using-extensions.md

```diff
@@ -56,7 +56,7 @@ If Windows Admin Center is installed on a computer that isn't connected to the i
 2. Click on the name of the package you want to install to view the package details page.
 3. Click on the **Download** link in the top navigation bar of the package details page and download the .nupkg file for the extension.
 4. Repeat steps 2 and 3 for all the packages you want to download.
-5. Copy the package files to a file share that can be accessed from the computer Windows Admin Center is installed on, or to the local disk of the computer.
+5. Copy the package files to a file share that can be accessed from the computer Windows Admin Center is installed on, or to the local disk of the computer on which Windows Admin Center is installed.
```

Comment:
I would have suggested "on which **the** Windows Admin Center is installed", but never mind.